codespan/src/span.rs

```diff
@@ -5,7 +5,7 @@ use std::ops::Range;
 
 use crate::ByteIndex;
 
-#[derive(Debug, Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
+#[derive(Debug, Default, Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
```

Comment:
Perhaps we could implement `Default` using `Span::intitial` in an explicit `impl`? I'm a tad uncomfortable leaving this up to a derive. 🤔

Reply:
Sure, that would probably be better. I'll modify the PR. Thanks for the suggestion!

Reply:
@brendanzab Committed, updated, and ready for re-review!

Reply:
Thanks!